commands/command_test.go

```diff
@@ -6,7 +6,199 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func (s *MmctlUnitTestSuite) TestCommandListCmdF() {
+func (s *MmctlUnitTestSuite) TestCommandCmdF() {
```

Comment:
All the tests related to a given command should be located inside a function with the command name, in this case:

```go
func (s *MmctlUnitTestSuite) TestCommandCreateCmdF() {
```